biomage/experiment/utils.py

```diff
@@ -116,3 +118,35 @@ def download_S3_json(s3_obj, key, filepath):
     s3_obj.download_file(local_file)
 
     set_modified_date(file_location=local_file, date=s3_obj.last_modified)
+
+
+def get_cognito_username(email):
+    client = boto3.client("cognito-idp")
+    user_name = client.admin_get_user(Username=email, UserPoolId="eu-west-1_sVQL15Yxu")[
```

Comment:
Looks to me that the userPoolId should be added to the config yaml instead: https://github.com/biomage-ltd/biomage-utils/blob/master/config.yaml

Reply:
Or maybe in constants.py?